dashboards/Analyze.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
so we can export all visualizations and dashboards and searches in 1 json file, I think we should just have that 1 file instead of multiple dashboard files